src/sacn/private/common.h

```diff
@@ -49,7 +49,8 @@ extern "C" {
 #define SACN_MTU 1472
 #define SACN_PORT 5568
 
-#define SACN_RECEIVER_MAX_SOCKET_REFS (((SACN_RECEIVER_MAX_UNIVERSES - 1) / SACN_RECEIVER_MAX_SUBS_PER_SOCKET) + 1)
+#define SACN_RECEIVER_MAX_SOCKET_REFS \
+  ((((SACN_RECEIVER_MAX_UNIVERSES - 1) / SACN_RECEIVER_MAX_SUBS_PER_SOCKET) + 1) * 2)
```

Comment:
What's the rationale behind adding the * 2 multiplier?

Reply:
The number of SocketRefs doubled when I added IPv6 support.

Reply:
I'll add a comment to clarify